inline-caching/cache-loader/build.gradle

```diff
@@ -0,0 +1,33 @@
+plugins {
+    id 'java'
+}
+
+group = 'com.vmware.gemfire.inlineCaching'
+version = '0.0.1-SNAPSHOT'
+sourceCompatibility = '1.8'
+
+
+task copyAllDependencies(type: Copy) {
+    from configurations.compileClasspath
+    include 'jooq-3.6.2.jar', 'HikariCP-4.0.3.jar'
+    into "${buildDir}/libs"
+}
+build.dependsOn(copyAllDependencies)
+
+repositories {
+    mavenCentral()
+    maven {
+        credentials {
+            username "$gemfireRepoUsername"
+            password "$gemfireRepoPassword"
+        }
+        url = uri("https://commercial-repo.pivotal.io/data3/gemfire-release-repo/gemfire")
+    }
+}
+
+dependencies {
+    implementation 'com.vmware.gemfire:geode-core:9.15.4'
```

Comment:
maybe this needs to be bumped to latest GemFire 10.0.x?

Reply:
this is the only version i could find at the time. I just looked again, how does one find versions of gemfire? I open up the [maven repo](https://commercial-repo.pivotal.io/repository/gemfire-release-repo/maven) but that was of no help either.

Reply:
there is 9.15.6 and 10.0.0

Reply:
ahh ok. i figured it out. and it works> :)